### ISSUE TITLE
ExampleObjectDetection not working

### ISSUE THREAD

**Issue by ToxaDev** [migueldeicaza/TensorFlowSharp#277]
Hi
I'm using master brunch and trying to launch ExampleObjectDetection example and got "Invalide GraphDef" runtime error
After some searching, i realized that this is due to the size of the buffer
And frozen_inference_graph.pb size is 186 Mb
How fix this error?

**Comment by migueldeicaza** [comment]
Did you run from the nuget packages?

**Comment by ToxaDev** [comment]
Yes, i opened the ExampleObjectDetection project and removed the reference to TensorFlowSharp. Then I added nuget version 1.7.0 and get the runtime error "Can not load DLL libtensorflow" But libtensorflow.dll presents in Debug folder..

After that i tried to use version 1.4.0 and now it works
It seems that the version of nuget 1.7.0 has a problem with loading libtensorflow.dll

**Comment by alex-zu** [comment]
> But get the runtime error "Can not load DLL libtensorflow" 
Please post detailed exception message (including HRESULT).

Keep in mind, libtensorflow.dll from nuget version 1.7.0 is x64 and apparently compiled with AVX support (if OS/processor not support AVX you may receive exception with HRESULT: 0xC000001D -  illegal instruction)

**Comment by bbday** [comment]
Better thing to do it is use gpu version dll (you can get it from python then rename it, use google for more info about it)

**Comment by migueldeicaza** [comment]
Mhm, I can not find a version of TEnsorFlow for Windows with GPU support.    You might need to compile your own version of TensorFlow and replace the binaries.
